Add troubleshooting for Telegram bot (#41239)
home-assistant/home-assistant.github.io: source/_integrations/telegram_bot.markdown

- Calls and live streaming
- Wallpapers and Themes

## Troubleshooting

{% details "Error sending message: Can't parse entities" %}

When using send actions such as `telegram_bot.send_message` with the `markdownv2` parse mode, the action will fail with the "Can't parse entities" error if the user input in the `message` field contains malformed Markdown syntax.

> You can perform any of the following to resolve this issue:

Reviewer: ```suggestion
You can perform any of the following steps to resolve this issue:
```

- Use the `plain_text` parse mode either by configuring the Telegram bot options or by specifying it via the action's `parse_mode` data attribute.
- Escape special characters in the `message` field with a preceding '\\' character.
- Format your message according to the [formatting options](https://core.telegram.org/bots/api#formatting-options).

{% enddetails %}

{% details "Telegram Webhook bot is unable to receive updates" %}

> If your Telegram bot is unable to receive updates (i.e. all events other than `telegram_sent` are not triggered), please follow the troubleshooting steps below:

Reviewer: ```suggestion
If your Telegram bot is unable to receive updates (for example, all events other than `telegram_sent` are not triggered), please follow the troubleshooting steps below:
```
